src/community/index.md

```diff
@@ -75,7 +75,6 @@ Learn how to
 Our wonderful community has provided these resources.
```

Comment:
```suggestion
Our wonderful community has provided these resources:
```
Idk if it's bad practice to add random punctuation changes to a PR with a specific purpose like this, but I just noticed a colon could be better here. Feel free to include or not!

Reply:
Doesn't hurt with such a small (and slightly related change)!